etna/articles/models.py

```diff
@@ -43,7 +43,9 @@ class ArticleIndexPage(BasePageWithIntro):
         use_json_field=True,
     )
 
-    new_label_end_date = datetime.now() - timedelta(days=21)
+    @cached_property
+    def new_label_end_date(self):
+        return datetime.now().date() - timedelta(days=21)
```

Comment:
Can you see any reason not to make this a simple `is_new` property that returns a boolean? That might make things a little easier to make use of in templates.

We should also move this behaviour to the mixin class so that we're adhering to DRY principles.